osquery/tables/networking/curl_certificate.cpp

```diff
@@ -19,6 +19,19 @@
 #include <openssl/x509.h>
 #include <openssl/x509v3.h>
 
+#ifdef WIN32
+#include <windows.h>
```

Comment:
This should be moved inside cmake/flags.cmake under `windows_common_link_options`

Reply:
done, two were already present, I added the third